jina/executors/encoders/audio/spectral.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+
+from .. import BaseAudioEncoder
+from ...decorators import batching, as_ndarray
+
+
+class MFCCTimbreEncoder(BaseAudioEncoder):
+    """
+    :class:`MFCCTimbreEncoder` is based on Mel-Frequency Cepstral Coefficients (MFCCs) which represent timbral features.
+    :class:`MFCCTimbreEncoder` encodes an audio signal from a `Batch x Signal Length` ndarray into a
+    `Batch x Concatenated Features` ndarray.
+    """
+
+    def __init__(self, input_sample_rate: int = 22050, n_mfcc=20, n_fft_length=2048, hop_length: int = 512, *args,
```

Comment:
```suggestion
    def __init__(self, input_sample_rate: int = 22050, n_mfcc: int = 20, n_fft_length: int = 2048, hop_length: int = 512, *args,
```
Please add typing information for the APIs.